src/main/java/com/adregamdi/like/application/LikesService.java

```diff
@@ -26,15 +27,21 @@ public class LikesService {
     private final LikesRepository likesRepository;
     private final LikesValidService likesValidService;
 
-    public void create(String memberId, CreateLikesRequest request) {
+    public CreateLikesResponse create(String memberId, CreateLikesRequest request) {
+
+        boolean isLiked = likesRepository.findByContentTypeAndContentId(request.getContentType(), request.contentId()).isPresent();
```

Comment:
조회 시 null 발생하면 원시 타입 boolean은 위험하지 않나요?? isPresent가 null이면 false로 처리해 주나요??

Reply:
```java
    /**
     * If a value is present, returns {@code true}, otherwise {@code false}.
     *
     * @return {@code true} if a value is present, otherwise {@code false}
     */
    public boolean isPresent() {
        return value != null;
    }
 ```
isPresent가 null 값에 대한 핸들링 기능을 담당해줍니다!!